pkgs/development/libraries/qt-5/modules/qtbase.nix

```
@@ -15,7 +15,7 @@
   # optional dependencies
 , cups ? null, postgresql ? null
 , withGtk3 ? false, dconf, gtk3
-, qttranslations ? null
+, withQttranslation ? true, qttranslations ? null
```

```suggestion
, withQttranslation ? true, qttranslations
```
if we have an option, we don't need the nullable